packages/arb-node-core/ethbridge/validator.go

```diff
@@ -212,7 +212,7 @@ func CreateValidatorWallet(
 	query := ethereum.FilterQuery{
 		BlockHash: nil,
 		FromBlock: big.NewInt(fromBlock),
-		ToBlock:   nil,
+		ToBlock:   big.NewInt(fromBlock + 1000),
```

Comment:
Good point.  I updated it default to query 1000 blocks at a time until log is found, can supply `--rollup.block-search-size=0` to allow searching all blocks at once in case L1 is a node provider with optimized log queries